pinot-spi/src/main/java/org/apache/pinot/spi/plugin/PluginManager.java

```diff
@@ -28,19 +28,89 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class PluginManager {
 
+  public static final String PLUGINS_DIR_ENV_VAR = "plugins.dir";
```

Comment:
we are looking for these in system property not env. Let's just call it PLUGINS_DIR_PROPERTY_NAME.